04-Coffee-and-Coding.Rmd

```diff
@@ -0,0 +1,44 @@
+# Coffee and Coding {#CC}
+
+## About Coffee and Coding
+
+The aim of the MoJ Coffee and Coding presentations is to nurture, enable and encourage a vibrant, supportive and inclusive coding community at MoJ. Coffee & Coding provides a regular opportunity for people within MoJ who are interested in coding to:
+
+* Share skills and knowledge
+* Showcase their work and projects
+* Provide advice and guidance to each other
+* Network and get to know each other
+
+Code can be shared from any language — whether R, Python, SQL, or other more specialised languages.
+
+The format is generally 30–40 minutes for a presentation, followed by time for questions. The presentations usually take the form of a demonstration of a tool or technique and/or a show and tell of work done within the department using these methods.
+
+## Recorded presentations
+
+Where possible, Coffee and Coding presentations are recorded to allow people to catch up on talks they may have missed and to provide a useful training resource for colleagues.
+
+These recordings are available on the [Coffee and Coding Microsoft Stream Channel](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630). Access should be granted to all new starters, but if you cannot access the videos please contact the Coffee and Coding mailbox.
+
+### Some useful talks
+
+If you are new to coding in MoJ, you might find the following recorded talks useful:-
+
+* [An Introduction to GitHub](https://web.microsoftstream.com/video/fa7c803c-129c-4b0d-bedf-7dd8edeef333?channelId=f6aa6c5d-e90c-44b7-8ccc-28a318fa0630)
+
+* [Ten Top Tips for Data Presentation](https://web.microsoftstream.com/video/393d138f-bb4d-449f-a707-fe122e8e7aa0)
+
+* [An Introduction to Airflow 2.0](https://web.microsoftstream.com/video/712b49b7-8995-4802-95de-c813c8b213bc)
+
+* [Package Management in renv](https://web.microsoftstream.com/video/3ec54ac3-473c-4268-9d54-9f7096338824)
+
+The full catalogue of recorded talks can be found [here](https://web.microsoftstream.com/channel/f6aa6c5d-e90c-44b7-8ccc-28a318fa0630).
+
+## Code repositories
+
+You can find code examples to go along with past Coffee and Coding presentations in our [private repo](https://github.com/moj-analytical-services/Coffee-and-Coding) (only accessible by members of the MoJ Analytical Services GitHub organisation) or [public repo](https://github.com/moj-analytical-services/Coffee-and-Coding-public).
+
+## Get involved!
+
+We are always looking for new presentations. If you have an idea for a talk you would like to give, or if you want to showcase a coding project you have been working on, please contact the Coffee and Coding mailbox.
```

Comment:
```suggestion
We are always looking for new presentations. If you have an idea for a talk you would like to give, or if you want to showcase a coding project you have been working on, please contact the Coffee and Coding mailbox: Coffee-and-Coding [at] justice.gov.uk.
```

Agreed, I'll put the @ symbol in brackets to avoid having the direct link.